### PR TITLE
Fixed bugs and typos which produced warnings

### PR DESCRIPTION
**Pull request by dukesrg**
This one should be merged after previous one. Will change binaries because actual bugs is fixed along with warnings suppression.
upd: Checked on O3DS9.2E - ninjhax, spiderhax, emunand, agb_firm (reverted to non-patched before updating from 2.5.2)

### CODE REVIEW

File: rxtools/source/lib/console.c [hunk 1]
```diff
@@ -100,7 +100,7 @@ void ConsoleShow(){
 	DrawString(tmpscreen, consoletitle, ConsoleX + BorderWidth + 2*CHAR_WIDTH, ConsoleY + (titlespace-CHAR_WIDTH)/2 + BorderWidth, TextColor, BackgroundColor);
 	
 	char tmp[256], *point;
-	if(findCursorLine < MAXLINES) point = &console[0];
+        if(findCursorLine() < MAXLINES) point = &console[0];
```
Comment (Syphurith) [inline rxtools/source/lib/console.c]: Anything abnormal due to this line?

File: rxtools/source/lib/ncch.c [hunk 2]
```diff
@@ -7,39 +7,39 @@ unsigned int align(unsigned int offset, unsigned int alignment)
...
+	unsigned long long n = 0;
+
+	n |= (unsigned long long)p[0]<<56;
+	n |= (unsigned long long)p[1]<<48;
+	n |= (unsigned long long)p[2]<<40;
+	n |= (unsigned long long)p[3]<<32;
+	n |= (unsigned long long)p[4]<<24;
+	n |= (unsigned long long)p[5]<<16;
+	n |= (unsigned long long)p[6]<<8;
+	n |= (unsigned long long)p[7]<<0;
```
Comment (Syphurith) [inline rxtools/source/lib/ncch.c]: well. so i should define a `u64` inside `common.h`..

File: rxtools/source/lib/console.c (hunk 1 above)
Comment (dukesrg) [inline rxtools/source/lib/console.c]: MAXLINES as for sure smaller than findCursorLine address, so condition never met.

**Comment by Syphurith** [comment]
There are quite some parts similar from the last one seen #48. I don't know if you're offering options for @roxas75 this time. If you would like to push commt after the pull is created is ok.

**Comment by dukesrg** [comment]
@Syphurith 
Previous pool request does contain nothing that actually change compilation result, only affects compiler warning generation, so may be merged blindly. This one contain the real code bug fixes.

**Comment by Syphurith** [comment]
@dukesrg Yes. No need to mention the "must-be" conflicts between these two? *Bye for sleep*

**Comment by dukesrg** [comment]
@Syphurith this one is based on a branch which is rebased from my master fork on which previous pool request was created. So it is a straight-forward merge :)

**Pull request merged**